### PR TITLE
[DOC] add Docker on Windows GPU infer instruction

### PR DESCRIPTION
**Pull request by generalova-kate**

### CODE REVIEW

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 1]
```diff
@@ -76,6 +76,62 @@ If you want to try some demos then run image with the root privileges (some addi
...
+    ~~~
+    where 
+    * `--device class/5B45201D-F2F2-4F3B-85BB-30FF1F953599` reserved GPU device interface class GUID
+    * `C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_518f2921ba495409` path to OpenCL driver home directory, please find it on your PC by `C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_*` regular expression.
+    *  `C:\tmp` folder with copy OpenCL.dll from your C:\Windows\System32 host folder
+
+2. Then copy OpenCL.dll to C:\Windows\System32 folder inside the container and set appropriate registry entry. Now you can run inference on GPU device:
+    ~~~
+    copy C:\tmp\OpenCL.dll C:\Windows\System32\ && reg add "HKLM\SOFTWARE\Khronos\OpenCL\Vendors" /v "C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_518f2921ba495409\ocl\bin\x64\intelocl64.dll" /t REG_DWORD /d 0
+    ~~~
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-docker-windows.md:129]:
```suggestion
2. Copy `OpenCL.dll` to the `C:\Windows\System32` folder inside the container and set appropriate registry entry. Now you can run inference on a GPU device:
   ~~~
   copy C:\tmp\OpenCL.dll C:\Windows\System32\ && reg add "HKLM\SOFTWARE\Khronos\OpenCL\Vendors" /v "C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_518f2921ba495409\ocl\bin\x64\intelocl64.dll" /t REG_DWORD /d 0
   ~~~
```

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 2]
```diff
@@ -76,6 +76,62 @@ If you want to try some demos then run image with the root privileges (some addi
...
+3. If you want to try demo_security_barrier_camera demo(some additional 3-rd party dependencies will be installed):
+    ~~~
+    cd bin && setupvars.bat && cd ../ && cd deployment_tools\demo && demo_security_barrier_camera.bat -d GPU -sample-options -no_show
+    ~~~
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-docker-windows.md:133]:
```suggestion
3. For example, run the `demo_security_barrier_camera` demo with the command below:  
   ~~~
   cd bin && setupvars.bat && cd ../ && cd deployment_tools\demo && demo_security_barrier_camera.bat -d GPU -sample-options -no_show
   ~~~
   > **NOTE**: Addittional third-party dependencies will be installed.
```

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 3]
```diff
@@ -76,6 +76,62 @@ If you want to try some demos then run image with the root privileges (some addi
 docker run -itu ContainerAdministrator --rm <image_name> cmd /S /C "cd deployment_tools\demo && demo_security_barrier_camera.bat -d CPU -sample-options -no_show"
 ~~~
 
+## Build and Run the Docker* Image for GPU
+
+GPU Acceleration in Windows containers feature requires to meet Windows host, OpenVINO toolkit and Docker* requirements:
+* [Windows requirements](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/gpu-acceleration)
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-docker-windows.md:82]:
```suggestion
* [Windows requirements](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/gpu-acceleration):
```

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 4]
```diff
@@ -76,6 +76,62 @@ If you want to try some demos then run image with the root privileges (some addi
...
+  * The container base image must be mcr.microsoft.com/windows:1809 or newer. Windows Server Core and Nano Server container images are not currently supported.
+  * The container host must be running Docker Engine 19.03 or newer.
+  * The container host must have a GPU running display drivers version WDDM 2.5 or newer.
+* [OpenVINO GPU requirement](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_windows.html#Install-GPU)
+  * Intel Graphics Driver for Windows version 15.65 or higher.
+* [Docker isolation mode requirement](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/hyperv-container)
+  *  Windows host and container version tags should be match.
+  *  [Windows host and container isolation process support](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility)
+
+##Build a Docker* Image for your host system
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-docker-windows.md:93]:
```suggestion
## Build a Docker* Image for Your Host System
```

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 5]
```diff
@@ -76,6 +76,62 @@ If you want to try some demos then run image with the root privileges (some addi
...
+
+1. You can re-use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles).
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-docker-windows.md:95]: Why do we say *can*? Is this step optional? Where else can the user get Dockerfiles?

**Review by aalborov — CHANGES_REQUESTED** [review]
There are problems with rendering lists and codeblocks in the [built version](http://openvino-docs.inn.intel.com/PR-3492/openvino_docs_install_guides_installing_openvino_docker_windows.html#build_and_run_the_docker_image_for_gpu). I've suggested a solution, but we'll need to rebuild once it's applied to make sure it works.

File: docs/install_guides/installing-openvino-docker-windows.md (hunk 5 above)
Comment (generalova-kate) [reply docs/install_guides/installing-openvino-docker-windows.md]: He can create himself Dockerfile or reuse our (it is prefer).

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 6]
```diff
@@ -76,6 +76,62 @@ If you want to try some demos then run image with the root privileges (some addi
...
+    copy C:\Windows\System32\OpenCL.dll C:\tmp
+    ~~~
+## Run the Docker* Image for GPU
+
+1. To try infer on GPU, run the image with the following command:
+    ~~~
+    docker run -it --rm -u ContainerAdministrator --isolation process --device class/5B45201D-F2F2-4F3B-85BB-30FF1F953599 -v C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_518f2921ba495409:C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_518f2921ba495409 -v C:\tmp:C:\tmp <image_name>
+    ~~~
+    where 
+    * `--device class/5B45201D-F2F2-4F3B-85BB-30FF1F953599` reserved GPU device interface class GUID
```
[the remark this answers is not in the document]
Comment (generalova-kate) [reply docs/install_guides/installing-openvino-docker-windows.md]: it is a common phrase from Windows docs. I can change it to  "reserved interface class GUID for GPU device". is it ok?

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 7]
```diff
@@ -76,6 +76,62 @@ If you want to try some demos then run image with the root privileges (some addi
...
+    * `C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_518f2921ba495409` path to OpenCL driver home directory, please find it on your PC by `C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_*` regular expression.
```
[the remark this answers is not in the document]
Comment (generalova-kate) [reply docs/install_guides/installing-openvino-docker-windows.md]: in the common search box. I think the developer can understand it without explonation.

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 8]
```diff
@@ -76,6 +76,64 @@ If you want to try some demos then run image with the root privileges (some addi
...
+   ~~~
+    
+## Run the Docker* Image for GPU
+
+1. To try inference on a GPU, run the image with the following command:  
+   ~~~
+   docker run -it --rm -u ContainerAdministrator --isolation process --device class/5B45201D-F2F2-4F3B-85BB-30FF1F953599 -v C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_518f2921ba495409:C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_518f2921ba495409 -v C:\tmp:C:\tmp <image_name>
+   ~~~  
+   where 
+   * `--device class/5B45201D-F2F2-4F3B-85BB-30FF1F953599` is a reserved GPU device interface class GUID.
```
Comment (generalova-kate) [inline docs/install_guides/installing-openvino-docker-windows.md:123]:
```suggestion
   * `--device class/5B45201D-F2F2-4F3B-85BB-30FF1F953599` is a reserved interface class GUID for GPU device.
```

File: docs/install_guides/installing-openvino-docker-windows.md (hunk 5 above)
Comment (aalborov) [reply docs/install_guides/installing-openvino-docker-windows.md]: Okay let's then have it like this:
```
1. Reuse one of [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles). You can also use your own Dockerfile.
```

**Review by aalborov — APPROVED** [review]
Everything looks good, the output is correct now! Please, apply that comment about available Dockerfiles.

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 9]
```diff
@@ -76,6 +76,64 @@ If you want to try some demos then run image with the root privileges (some addi
...
+  * The container host must have GPU running display drivers of version WDDM 2.5 or higher.
+* [OpenVINO™ GPU requirement](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_windows.html#Install-GPU):
+  * Intel Graphics Driver for Windows of version 15.65 or higher.
+* [Docker isolation mode requirement](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/hyperv-container):
+  *  Windows host and container version tags must match.
+  *  [Windows host and container isolation process support](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility)
+
+## Build a Docker* Image for Your Host System
+
+1. You can reuse [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles).
```
Comment (generalova-kate) [inline docs/install_guides/installing-openvino-docker-windows.md:95]:
```suggestion
1. Reuse one of [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles). You can also use your own Dockerfile. 
```

**Pull request merged**